policystorage/nomad.go

```diff
@@ -74,7 +74,7 @@ func canonicalize(from *api.ScalingPolicy, to *Policy) {
 	to.Target.Config["group"] = from.Target["Group"]
 
 	if to.Source == "" {
-		to.Source = "local-nomad"
+		to.Source = "nomad-apm"
```

Comment:
Same as before. Have this value in a single place and just reference it in other places.